index.js

```diff
@@ -0,0 +1,53 @@
+var map = document.querySelector(".contacts-map");
+var modalmap = document.querySelector(".modal-map");
+var close = document.querySelector(".map-close");
+            
+var writeus = document.querySelector(".writeus-button");
+var modalform = document.querySelector(".modal-write");
+var fio = modalform.querySelector("[name=user-name]");
+var formclose = document.querySelector(".write-close");
+var form = modalform.querySelector("form");
+var email = modalform.querySelector("[name=user-email]");
+
+
+map.addEventListener("click", function (evt) {
+    evt.preventDefault();
+    modalmap.classList.add("map-show");
+});
+close.addEventListener("click", function (evt) {
+    evt.preventDefault();
+    modalmap.classList.remove("map-show");
+});
+window.addEventListener("keydown", function(evt) {
+    if (evt.keyCode === 27) {
+        if (modalmap.classList.contains("map-show")) {
+            evt.preventDefault();
+            modalmap.classList.remove("map-show");
+        }
+    }
+});           
+            
+writeus.addEventListener("click", function(evt) {
+    evt.preventDefault();
```

Comment:
here it is reasonable, bc you don't make default action

Reply:
please rething preventDefaults below)

Reply:
do you mean that evt have to be the first line after function?

Reply:
I meant here usage of preventDefault is needed)